packages/web-runtime/src/pages/account.vue

```diff
@@ -134,18 +126,15 @@ export default defineComponent({
       return null
     },
     groupNames() {
-      return this.groups.join(', ')
+      if (this.capabilities.spaces?.enabled) {
```

Comment:
Can't we simply do a type check? if already array of strings -> just join them. If objects -> extract displayName.

Reply:
https://github.com/owncloud/web/pull/7568/commits/f82c21fcbecfa02ec9a8713c89ab45b1e6359716

To be honest, I am not a fan of duck typing, using the capability makes the code more self-explanatory, especially when the get the graph capability